utils/setup/docstring_check_result.txt

```diff
@@ -0,0 +1,1145 @@
+************* Module baselines.marl_benchmark.run
+baselines/marl_benchmark/run.py: C0114: line 1: Missing module docstring (missing-module-docstring)
+baselines/marl_benchmark/run.py: C0116: line 36: Missing function or method docstring (missing-function-docstring)
+baselines/marl_benchmark/run.py: C0116: line 95: Missing function or method docstring (missing-function-docstring)
```

Comment:
Also I've ignored these directories in the latest change. But please note that `pylint` currently only accept names rather than pathes. In other words, if there are multiple directories named `marl_benchmark`, `examples` ..., all of them and their sub-directories would be ignored. I've checked that for example, there really are multiple `examples` folders in different directories, do we want to exclude all of them?

Reply:
> I think we should discuss with the team whether we should enforce module docstrings.

I've removed the module check, but if necessary, we can bring it back easily.

Reply:
> But please note that pylint currently only accept names rather than pathes. In other words, if there are multiple directories named marl_benchmark, examples ..., all of them and their sub-directories would be ignored.

Could you clarify what you mean by this? Have you come across some documentation on it?